.github/workflows/e2e-testnet.yml

```diff
@@ -0,0 +1,45 @@
+name: E2E tests / Testnet
+
+on:
+  schedule:
+    - cron: '*/20 * * * *'
+  workflow_dispatch:
+  pull_request: # TODO: remove this trigger before merging to main
```

Comment:
Can we remove it yet?

Reply:
We need to keep it as long as we need to test the workflow (cron and workflow_dispatch won't work until the workflow is merged to `main`). I've added the TODO in the PRs description in order not to forget about removing this once we're ready.

Reply:
Replaced it with a cron which runs this job four times a day: https://github.com/keep-network/local-setup/pull/98/commits/152e9f00514a7066d646d0c4bf72f36c61df2e77